.swiftpm/xcode/xcshareddata/xcschemes/AWSAPIPlugin.xcscheme

```diff
@@ -20,6 +20,20 @@
                ReferencedContainer = "container:">
             </BuildableReference>
          </BuildActionEntry>
+         <BuildActionEntry
+            buildForTesting = "YES"
+            buildForRunning = "YES"
+            buildForProfiling = "NO"
+            buildForArchiving = "NO"
+            buildForAnalyzing = "NO">
+            <BuildableReference
+               BuildableIdentifier = "primary"
+               BlueprintIdentifier = "AWSAPIPluginTests"
+               BuildableName = "AWSAPIPluginTests"
+               BlueprintName = "AWSAPIPluginTests"
+               ReferencedContainer = "container:">
+            </BuildableReference>
+         </BuildActionEntry>
```

Comment:
what is this for? do we need it?

Reply:
I think these are the schema changes when I move the target to `AWSAPIPluginTests`, shall I discard these changes ?

Reply:
This looks like it is under the AWSAPIPlugin scheme, which should build the AWSAPIPlugin target with CMD+B and execute the tests using AWSAPIPluginTests with CMD+U. Looks like AWSAPIPluginTests is added and set to "build for running" which looks inaccurate. I would discard this change and see if you need anything else to achieve the development workflow of building and testing AWSAPIPlugin, which i think was already set up.

Reply:
oh sure, discarded the changes